.service

```
@@ -0,0 +1,15 @@
+[Unit]
+Description=Wicked migration server
+After=network.target
+
+[Service]
+User=root
+Group=root
+ExecStart=wicked_migration_server -i :: -p 8080 -s /srv/wicked_migration_server/html -d /var/lib/wicked_migration_server/db.db3
+Restart=always
+RestartSec=5
+StandardOutput=journal
+StandardError=journal
```

```suggestion
ExecStart=wicked_migration_server -i :: -p 8080 -s /srv/wicked_migration_server/html -d /var/lib/wicked_migration_server/db.db3
Restart=always
RestartSec=5
```
In general I would for systemd service files keep them as minimal as possible to apply the defaults. And these options should already be the default.